content/doc/developer/forms/jelly-form-controls.adoc

```diff
@@ -121,3 +133,30 @@ Then, use an `<f:entry>` tag to enclose the normal select tag.
    <f:select />
 </f:entry>
 ----
+
+== Inline help
+
+* jelly definition link:https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/helpLink.jelly[form/helpLink.jelly]
+
+Most of form elements allows to have an inline help generated by adding a small html file and following conventions.
+
+For example <f:entry> tag, uses the field name to find the
```

Comment:
```suggestion
For example the `<f:entry>` tag uses the `field` attribute.
```

(not sure about exotic AsciiDoc syntax in the first case, may need `+` or `++` or something)